cg/store/filters/status_case_filters.py

```diff
@@ -111,18 +111,36 @@ def filter_case_by_internal_id(cases: Query, internal_id: str, **kwargs) -> Quer
     return cases.filter(Family.internal_id == internal_id)
 
 
+def filter_cases_by_customer_entry_id(cases: Query, customer_entry_id: int, **kwargs) -> Query:
+    """Return cases with matching customer id."""
+    return cases.filter(Customer.id == customer_entry_id)
```

Comment:
Should we start refactoring entry id to database id?

Reply:
Not sure, maybe we can do that later in separate PR:s. I think I prefer customer_entry_id.

Reply:
A quick googling seems to indicate that entry id is a good name for a unique database id. Let's keep it and use it consistently. 👍

Reply:
Ok!